editor/src/editor/UserInterface.cpp

```diff
@@ -262,6 +262,10 @@ void UserInterface::update(graphics::Graphics& graphics, float time_elapsed)
                     gui_.get("character_object_window")->setVisible(false);
                     gui_.get("obstacle_object_window")->setVisible(false);
                 }
+                else if (event.key.code == sf::Keyboard::Q)
```

Comment:
Maybe ESC key? Or Delete? In my opinion it will be more intuitive. Anyway, after #242 is closed it won't matter.